### PR TITLE
Merge/group merge

### PR DESCRIPTION
**Pull request by peterkimpk1**
# Pull Request
## Description
- Show component renders specific TicketMaster shows based on artist selection
- Added keyframe animations for artistCard/login
- Created mock server using mirage.js 
- Added extra types in type file
## Related Issues
- CreateEventForm 
## Changes Made
- ArtistCards, ArtistShows, CreateEvent
## Testing
- Need to test user flow
## Checklist
- [x] The code follows the project's coding standards.
- [x] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [ ] The code compiles without errors.
- [x] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

## Contributor(s)
- Peter Kim, Adam Konber, Jarvis Bailey, Kyle Boomer

### CODE REVIEW

**Review by baileyjarvis2814 — COMMENTED** [review]
I primarily worked on styling and adding some primary logic for adding friends of the logged in user.

File: src/App.css
Comment (baileyjarvis2814) [inline src/App.css:1]: Added background image to be used in app

File: src/Components/CreateEvent/CreateEvent.css
Comment (baileyjarvis2814) [inline src/Components/CreateEvent/CreateEvent.css:1]: Added keyframes for createevent card for styling purposes

File: src/Components/FriendsList/FriendsList.css
Comment (baileyjarvis2814) [inline src/Components/FriendsList/FriendsList.css:1]: Added key frames for friend card to fade and slide in.

File: src/Components/FriendsList/FriendsList.tsx
Comment (baileyjarvis2814) [inline src/Components/FriendsList/FriendsList.tsx:1]: Prepared logic to pass in logged in user as a prop, the filter out logged in user and add other users as friends of logged in user

**Comment by Sterling47** [comment]
Helped jarvis add revamp styles to components

File: src/App.tsx [hunk 1]
```diff
@@ -11,28 +11,92 @@ import EventPage from './Components/EventPage/EventPage.js'
...
 const AuthContext =React.createContext<boolean>(false);
 
+
+
+const EventModel: ModelDefinition<Event> = Model.extend({})
+type AppRegistry = Registry<
+  {event: typeof EventModel;},
+  {events: typeof EventModel;}
+>;
+new Server<AppRegistry>({
```
Comment (peterkimpk1) [inline src/App.tsx:28]: Creates a new mock server with routes/seeds using mirage.js library

File: src/Components/LoginPage/LoginPage.css
Comment (baileyjarvis2814) [inline src/Components/LoginPage/LoginPage.css:1]: Changed overall styling to make it uniform with Adam's

File: src/assets/concertmate.jpg
Comment (baileyjarvis2814) [inline src/assets/concertmate.jpg:1]: Added background image

**Comment by peterkimpk1** [comment]
- Worked on creating the mock server using mirage.js, following JSON contract from backend for get/post events

File: src/Components/CreateEventForm/CreateEventForm.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const {id} = user
+    const tmData = tmData1[id-1]
+    let ticketmaster_event_id;
+    
+    let singleArtistShowsData = tmData.find(artist => artist.name === selectedArtist)
+    if (!singleArtistShowsData.url.includes("ticketmaster")) {
+      
+    }
+    
+    const userEvent = {
```
Comment (Sterling47) [inline src/Components/CreateEventForm/CreateEventForm.tsx:29]: Looks like we need to finish userEvents body inputs

File: src/Components/EventCard/EventCard.css
Comment (Sterling47) [inline src/Components/EventCard/EventCard.css:1]: revamp event card

**Review by KyleMBoomer — APPROVED** [review]
Almost too much going on to leave a conscientious remark, but we reviewed/designed most of these changes live on a call together. 
-Consolidation of mock data looks good
-Initial styling looks a lot more sleek and polished, love the Rebecca purple
-New TS types requisite for our components to function properly are initiated in our type.ts file and imported properly
-ArtistShows component functioning as expected, and nested inside CreateEvent as planned
Now we need to implement the Add Friend feature to an event, and see it post to our mock server when Create Event is clicked by the user. After that, testing the user flow with Cypress should conclude our requisite needs to meet MVP for Part I. Great work, everyone!

**Pull request merged**